docs/source/guides/setting_config_options.ipynb

```diff
@@ -140,9 +140,9 @@
     "\n",
     "Woodwork provides the option to infer string columns as the `EmailAddress` logical type if a representative sample of valid (non-missing) rows all match a given regular expression. The `email_inference_regex` config variable allows users to set the regular expression that is used during this matching process. The default regex is `r\"(^[a-zA-Z0-9_.+-]+@[a-zA-Z0-9-]+\\.[a-zA-Z0-9-.]+$)\"` (taken from https://emailregex.com/).\n",
     "\n",
-    "### URL Inference Regex\n",
+    "### IP Address Inference Regex\n",
     "\n",
-    "Woodwork provides the option to infer string columns as the `URL` logical type if a representative sample of valid (non-missing) rows all match a given regular expression. The `url_inference_regex` config variable allows users to set the regular expression that is used during this matching process. The default regex is `r\"(http[s]?://(?:[a-zA-Z]|[0-9]|[$-_@.&+]|[!*\\(\\),]|(?:%[0-9a-fA-F][0-9a-fA-F]))+)\"` (taken from https://urlregex.com/)."
+    "Woodwork provides the option to infer string columns as the `IPAddress` logical type if a representative sample of valid (non-missing) rows all match a given regular expression. The `ipv4_inference_regex` and `ipv6_inference_regex` config variables allow users to set the regular expression that is used during this matching process. The default for `ipv4_inference_regex` is taken from https://ipregex.com/ and the default for `ipv6_inference_regex` is taken from https://regex101.com/r/cT0hV4/5."
```

Comment:
@ajaypallekonda Are you sure about the source for the ipv6 regex? I found some better ones
- https://www.oreilly.com/library/view/regular-expressions-cookbook/9781449327453/ch08s17.html
- https://ihateregex.io/expr/ipv6/
- https://gist.github.com/dfee/6ed3a4b05cfe7a6faf40a2102408d5d8 (this one is based on yours, but improves on it, though a comment mentions one mistake)

Reply:
I changed the ipv6 regex to this one: https://ihateregex.io/expr/ipv6/